internal/server/v1/observations/golden/bulk_point/all_latest.json

```diff
@@ -395,8 +395,8 @@
               "facet": "2978659163"
             },
             {
-              "date": "1989-12",
-              "value": 1144.084211,
+              "date": "2022-11",
```

Comment:
This is from imputation, but the date gap seems too big. Also Now the imputation result could be returned to the client. Do we want to have more UI control before showing them?

@jehangiramjad